webrtc-stats.html

```diff
@@ -917,6 +921,41 @@ <h2>
                   sender side. If no <a>RTCP Receiver Report</a> has been received yet, then return 0.
                 </p>
               </dd>
+              <dt>
+                <dfn>packetsReceivedWithEct1</dfn> of type <span class="idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP packets received for this <a>SSRC</a> marked with the "ECT(1)" marking.
+                </p>
+              </dd>
+              <dt>
+                <dfn>packetsReceivedWithCe</dfn> of type <span class="idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP packets received for this <a>SSRC</a> marked with the "CE" marking.
+                </p>
+              </dd>
+              <dt>
+              <dt>
+                <dfn>packetsReportedAsLost</dfn> of type <span class="idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP packets for which an [[RFC8888]] section 3.1 report has been sent with a zero R bit.
+                  Only reported if support for the "ccfb" feedback mechanism has been negotiated.
+                </p>
+              </dd>
+                <dt><dfn>packetsReportedAsLostButRecovered</dfn> of type <span class="idlMemberType">unsigned long long</span></dt>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP packets for which an [[RFC8888]] section 3.1 report has been sent with a zero R bit,
+                  but a later report for the same packet has the R bit set to 1.
+                  Only reported if support for the "ccfb" feedback mechanism has been negotiated.
+                </p>
+              </dd>
```

Comment:
should we change the name from *ButRecovered to *ButReceivedLater because these packets were perhaps received late not recovered by retx or fec.